apps/explorer/src/ui/stories/ProgrammableTransactionCommands.stories.tsx

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import {
+    type MoveCallSuiCommand,
+    type SuiArgument,
+    type SuiMovePackage,
+} from '@mysten/sui.js';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { BrowserRouter } from 'react-router-dom';
+
+import { RpcClientContext } from '../../../../core';
+
+import {
+    Command,
+    CommandType,
+    type ICommandProps,
+} from '~/pages/transaction-result/programmable-transaction-view/Command';
+import { DefaultRpcClient, Network } from '~/utils/api/DefaultRpcClient';
+
+export default {
+    title: 'ProgrammableTransaction',
+    component: Command,
+    decorators: [
+        (Story) => (
+            <BrowserRouter>
+                <RpcClientContext.Provider
+                    value={DefaultRpcClient(Network.LOCAL)}
+                >
+                    <Story />
+                </RpcClientContext.Provider>
+            </BrowserRouter>
+        ),
+    ],
+} as Meta;
+
+export const SplitCoin: StoryObj<ICommandProps<SuiArgument[]>> = {
+    args: { type: CommandType.SplitCoin, data: ['GasCoin', { Input: 1 }] },
+};
+
+export const TransferObjects: StoryObj<
+    ICommandProps<(SuiArgument | SuiArgument[])[]>
+> = {
+    args: {
+        type: CommandType.TransferObjects,
+        data: [
+            [
+                {
+                    Result: 0,
+                },
+                {
+                    Result: 1,
+                },
+                {
+                    Result: 2,
+                },
+                {
+                    Result: 3,
+                },
+                {
+                    Result: 4,
+                },
+            ],
+            {
+                Input: 0,
+            },
+        ],
+    },
+};
+
+export const MergeCoins: StoryObj<
+    ICommandProps<(SuiArgument | SuiArgument[])[]>
+> = {
+    args: {
+        type: CommandType.MergeCoins,
+        data: [
+            {
+                Input: 0,
+            },
+            [
+                {
+                    Result: 0,
+                },
+                {
+                    Result: 1,
+                },
+                {
+                    Result: 2,
+                },
+                {
+                    Result: 3,
+                },
+                {
+                    Result: 4,
+                },
+            ],
+        ],
+    },
+};
+
+export const Publish: StoryObj<ICommandProps<SuiMovePackage>> = {
+    args: {
+        type: CommandType.Publish,
+        data: {
+            disassembled: {
```

Comment:
Can we just make this something simpler?

Reply:
remove this for now since we do not render the display of Publish